src/nac/management/commands/import_devices_from_csv.py

```diff
@@ -93,35 +113,66 @@ def handle_deviceObject(self, deviceObject):
             self.save_invalid_devices(deviceObject)
         except Exception as e:
             logging.error(f"Error: Handling device Object failed -> {e}")
+            traceback.print_exc()
+
+    def get_deviceRole(self, deviceObject):
+        DeviceRoleCriteria = self.csv_mapping['DeviceRoleCriteria']
+        if DeviceRoleCriteria['MAPPING']:  # True -> DeviceRoles based on OU-Mapping, False -> DeviceRoles based on CSV-Mapping
```

Comment:
The comment could be removed, if you put the content into a function with a corresponding name. (see my comment above)